joomla/joomla-cms
# Add privacy plugin for exporting consent data, enable actionlogs privacy plugin by default

Pull Request for Issue #22708

### Summary of Changes

- Adds a privacy plugin to export user data from the `#__privacy_consents` table
    - This does not add the remove logic that would close #22720
- Enables the "Privacy - Action Logs" plugin by default, there is no reason it shouldn't be

### Testing Instructions

- Apply patch, ensure new plugin is installed and enabled, log a consent for the user who you will eventually be exporting data for, create an export request for that user, export their data, check the XML includes the consents

### Expected result

Consents included in export

### Actual result

Consents not included in export

### Documentation Changes Required

N/A, we don't have each individual plugin documented

### Stuck Between A Rock And A Hard Place

This is introducing a new plugin well after feature and language freeze for 3.9.  So we have a few options:

- Ship it in 3.10 (SemVer)
- Ship it in 3.9.1 (not SemVer but should be in core anyway)
- Ship it in 3.9.0 without translation requirement
